config/eslint/node.cjs

```diff
@@ -1,16 +1,25 @@
-function defaults(config) {
+function defaults(config = {}) {
   const result = {
-    files: !config?.useModules ? ['./babel.config.js', './.eslintrc.cjs', './index.js', './addon-main.cjs', './addon-main.js'] : [],
+    files: !config.useModules
+      ? ['./babel.config.js', './.eslintrc.cjs', './index.js', './addon-main.cjs', './addon-main.js']
+      : [],
     parserOptions: {
-      sourceType: config?.useModules ? 'module' : 'script',
+      sourceType: config.useModules ? 'module' : 'script',
       ecmaVersion: 2022,
     },
     env: {
       browser: false,
       node: true,
       es6: true,
+      ...config.env,
+    },
+    globals: config.globals || {},
+    plugins: ['n', 'import'],
+    extends: 'plugin:n/recommended',
+    rules: {
+      'import/order': ['error', { 'newlines-between': 'always' }],
```

Comment:
Should we always set `newlines-between` to `always`?

Reply:
if we can match the main library code pattern we should, there we do newlines between groups

Reply:
AFAICT this rule is redundant. Removing it. (Was just trying it out based on `._eslintrc`)